src/table.ts

```diff
@@ -210,24 +210,27 @@ class Table {
       PartialResultStream {
     const proxyStream = through.obj();
 
-    this.database.getSnapshot(options, (err, snapshot) => {
+    this.database.getSnapshot(options, (err: Error|null, snapshot?: Snapshot|null) => {
```

Comment:
Why do we need to do this? Are these types inferred?

Reply:
Nope, they aren't inferred